app/views/posts/show.html.erb

```diff
@@ -1,2 +1,12 @@
-<h1>Posts#show</h1>
-<p>Here is a single Post for a given user you are in '/users/user_id/posts'</p>
\ No newline at end of file
+<% content_for :title, "#{@user.name}'s Posts"%>
+<%= render "posts/user_post", post: @post %>
+<div class= "post-comments mt-4 border-2 border-gray-400">
+  <% @post.comments.each do |comment| %>
+    <div class="post-comment">
+      <p>
+        <span><strong><%= comment.author.name%></strong></span>
+        <span><%= comment.text %></span>
+      </p>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Similarly, like the previous reviewer mentioned,  you need to make this content center aligned by adding an auto margin to the left and right. Best of luck